lms/djangoapps/grades/signals/handlers.py

```diff
@@ -92,8 +92,8 @@ def score_published_handler(sender, block, user, raw_earned, raw_possible, only_
     if only_if_higher:
         previous_score = get_score(user.id, block.location)
 
-        if previous_score:
-            prev_raw_earned, prev_raw_possible = previous_score  # pylint: disable=unpacking-non-sequence
```

Comment:
FYI @sanfordstudent I was able to remove the same pylint warning from this other call as well.  Once again, by making it clear to pylint that the unpacking wouldn't be called unless the score wasn't `None` (it must be interpreting that from its code analysis in parsing the implementation of `get_score`.